.github/workflows/test.yml

```diff
@@ -134,65 +138,71 @@ jobs:
         run: |
           go test -race -shuffle=on -v -count=10 ./...
 
-  freebsd:
+  bsd:
     strategy:
       matrix:
+        os: ['FreeBSD', 'NetBSD']
         go: ['1.18.10', '1.19.13', '1.20.14', '1.21.13', '1.22.12', '1.23.6', '1.24.0']
-    name: Test with Go ${{ matrix.go }} on FreeBSD
+    name: Test with Go ${{ matrix.go }} on ${{ matrix.os }}
     runs-on: ubuntu-22.04
     defaults:
       run:
         shell: bash
     steps:
       - uses: actions/checkout@v4
       - name: Run in freebsd
+        if: matrix.os == 'FreeBSD'
         uses: vmactions/freebsd-vm@v1
         with:
           usesh: true
           prepare: |
             fetch https://go.dev/dl/go${{matrix.go}}.freebsd-amd64.tar.gz
             rm -fr /usr/local/go && tar -C /usr/local -xf go${{matrix.go}}.freebsd-amd64.tar.gz
             ln -s /usr/local/go/bin/go /usr/local/bin
-          run: |
-            # FreeBSD tests run within QEMU on Ubuntu.
+          run: &shared_run |
+            # FreeBSD  & NetBSD tests run within QEMU on Ubuntu.
```

Comment:
Remove one space char before &. (or just BSD)

Reply:
Just learned GitHub doesn't support anchors so the tests didn't run :(

Reply:
Wouldn't it be possible to create a shell script file and `source` it?